setup.cfg

```diff
@@ -33,3 +33,4 @@ scripts =
 
 [options.extras_require]
 latex2html =  pdf2image; Pillow
+cornellInterface = console-menu; bullet
```

Comment:
I am not very familiar with making installation and requirement files - how should platform-specific modules be listed?

Reply:
This is correct, but needs to be updated with os-specific logic. I believe it should look like this:
```
latex2html =  pdf2image; Pillow
interface = 
    console-menu; platform_system=="Windows"
    bullet; platform_system=="Linux" or platform_system=="Darwin"
```

See very bottom of: https://setuptools.readthedocs.io/en/latest/userguide/declarative_config.html?highlight=options.extras_require#configuring-setup-using-setup-cfg-files

I would also recommend switching the extra name from cornellInterface to just interface (as in the example above) to make the full install command shorter and easier to parse.  Also, we'll need to update the documentation with this accordingly.

Reply:
How would you prefer the documentation show examples for the extras? Should there be separate examples for latex2html and interface, or combine them into one and say the chosen optional dependencies are separated by commas within the brackets?